### DISCUSSION TITLE
Dev Container Feature to install Task

### DISCUSSION THREAD

**Discussion by eitsupi** [go-task/task#918] [Show and tell]
Hello. I often want to install this great tool in a container, so I tried to make it installable as a [Dev Container Feature](https://containers.dev/implementors/features/).
This may be useful for users using VSCode Dev Containers or GitHub Codespaces.

```json
{
	"name": "Debian",
	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
	"remoteUser": "vscode",
	"features": {
		"ghcr.io/eitsupi/devcontainer-features/go-task:latest": {}
	}
}
```

I am new to Task, so if you have any suggestions for improvement, please let me know in the repository.
https://github.com/eitsupi/devcontainer-features

**Comment by andreynering** [comment]
I didn't know about devcontainers, but this looks useful.

Thanks!
